sdk/tests/js/tests/compressed-texture-utils.js

```diff
@@ -192,6 +192,28 @@ var testTexSubImageDimensions = function(gl, validFormats, expectedByteLength, g
     gl.deleteTexture(tex);
 };
 
+var testTexImageLevelDimensions = function(gl, validFormats, expectedByteLength, getBlockDimensions, imageConfigs) {
```

Comment:
Might be a good time to start changing "var" to "let", at least inside this function.